xcs-ui/xcs-ui-mainwikipreferences/pom.xml

```diff
@@ -94,6 +94,20 @@
               <xpath>/xwikidoc/object/property/version</xpath>
               <value>${xcs.footer.version}</value>
             </transformation>
+            <!-- Specify different rights for XWiki.XWikiGuest -->
+            <transformation>
+              <artifact>org.xwiki.enterprise:xwiki-enterprise-ui-admin-user</artifact>
+              <file>XWiki/XWikiPreferences.xml</file>
+              <xpath>xwikidoc/object[property/users[text()="XWiki.XWikiGuest"]]/property/levels</xpath>
+              <value>none</value>
```

Comment:
I don't know how to reset the value to be empty. I've tried:
- " " 
 - Didn't worked - used the default values
- "<![CDATA[]]>" 
 - Didn't worked - used the default values
- "random_word" 
 - Works